packages/components/package.json

```diff
@@ -34,22 +36,27 @@
     "devDependencies": {
         "@frontify/eslint-config-react": "0.17.0",
         "@tailwindcss/forms": "^0.5.7",
+        "@testing-library/cypress": "^10.0.1",
         "@testing-library/jest-dom": "^6.4.2",
         "@testing-library/react": "^14.2.1",
         "@types/node": "^20.11.19",
         "@types/react": "^18.2.55",
         "@vitejs/plugin-react": "^4.2.1",
         "@vitest/coverage-v8": "^1.3.0",
         "@vitest/ui": "^1.3.0",
+        "cypress": "13.6.6",
+        "cypress-real-events": "1.12.0",
         "eslint": "^8.56.0",
         "eslint-plugin-notice": "^0.9.10",
         "happy-dom": "^13.3.8",
         "postcss": "^8.4.35",
         "postcss-import": "^16.0.1",
         "prettier": "^3.2.5",
         "react": "^18.2.0",
+        "react-dom": "^18.2.0",
         "tailwindcss": "^3.4.1",
         "typescript": "^5.3.3",
+        "vite": "^5.1.3",
```

Comment:
@noahwaldner we might need a tool to synchronize the dev dependencies and tooling between packages to avoid battling different versions at the same point in time.

In the past, I've used https://www.npmjs.com/package/syncpack and was quite happy with it, but we can consider other alternatives.

Reply:
let's talk about thhis!